docs/playroom/snippets.ts

```diff
@@ -640,14 +640,25 @@ const snippets: Array<Snippet> = [
 		group: 'ProgressIndicator',
 		name: 'Basic',
 		code: `<ProgressIndicator
+			activePath="/organisations/change-name"
 			items={[
-				{ label: 'Introduction', status: 'done', href: '#' },
-				{ label: 'Submit evidence', status: 'saved', href: '#' },
-				{ label: 'Organisations', status: 'started', href: '#', isActive: true },
-				{ label: 'Business contacts', status: 'error', href: '#' },
-				{ label: 'Case studies', status: 'todo', href: '#' },
-				{ label: 'Attachments', status: 'started', href: '#' },
-				{ label: 'Review and submit', status: 'blocked', href: '#' },
+				{ label: 'Introduction', status: 'done', href: '/introduction' },
+				{ label: 'Submit evidence', status: 'saved', href: '/evidence' },
+				{
+					label: 'Organisations',
+					status: 'started',
+					href: '/organisations',
+					items: [
```

Comment:
If we go with the word "items" I think it should be called `subItems`.

Reply:
I'm ok with `subItems`. Thoughts @stowball ?

Reply:
I kinda feel like items are items, and devs should be able to have the same object structure at whatever level they're at

Reply:
`items` matches SideNav

Reply:
`items` seems the go from our discussion.